introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    prediction_length = 14 * 24
</pre><p>I think this "magic number" warrants an inline comment even though you do explain above too. Suggested:</p><pre class="ql-syntax" spellcheck="false">prediction_length = 14 * 24 # 14 days x 24 hours = 2 weeks
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>